packages/pkgpanda-api/extra/pkgpanda-api

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+mkdir -p $WORK_DIR
+
+gunicorn --worker-class=sync \
```

Comment:
Can we please just put this in the systemd file as a bash command?

Doing wrapper scripts like this messes up some process tracking (Esp. since we aren't doing an exec here).